src/utils/ingest-client/ingest-file-validator.ts

```diff
@@ -30,22 +31,39 @@ class IngestFileValidator {
      * @param fileName
      */
     validateFile(fileResource: any, fileFormat: string, fileName: string) : Promise<ValidationJob> {
-        return this.assertNotAlreadyValidated(fileResource).then(fileChecksums => {
-            return this.uploadAreaForFile(fileResource).then(uploadAreaUuid => {
-                const validationImage = this.imageFor(fileFormat);
-                if(! validationImage) {
-                    return Promise.reject(new NoFileValidationImage());
-                } else {
-                    const imageUrl = validationImage.imageUrl;
-                    return IngestFileValidator._validateFile(fileName, uploadAreaUuid, imageUrl, this.uploadClient).then(validationJobId => {
-                        return Promise.resolve({
+
+        const validationJob = fileResource["validationJob"];
+        const fileChecksums = fileResource["checksums"];
+
+        if (validationJob) {
+            const completed = validationJob.jobCompleted;
+            if (!completed) {
+                return Promise.reject(new FileCurrentlyValidatingError());
+            }
+
+            if (validationJob.validationReport) {
+                const fileSha1 = fileChecksums.sha1;
+                const validatedSha1 = validationJob.checksums.sha1;
+
+                if (validationJob.validationReport.validationState == 'VALID' && fileSha1 == validatedSha1)
```

Comment:
we can be a bit more flexible here with the condition that triggers revalidation.